webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc

```diff
@@ -0,0 +1,34 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+Organizations looking to standardize their development environment can do so by adopting devfiles.
+In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If your organization needs custom devfiles that are authored and shared internally, then you will need a role based approach so developers, devfile authors, and registry administrators can interact together.
```

Comment:
I think we can avoid future tense since we've already specified a condition (if your organization needs custom devfiles). So, 
`If your organization needs custom devfiles that are authored and shared internally, then you need a role based approach...`

Reply:
updated